scripts/rollup/results.json

```diff
@@ -33,24 +33,24 @@
       "gzip": 29313
     },
     "react.development.js (NODE_DEV)": {
-      "size": 71914,
-      "gzip": 18276
+      "size": 72595,
+      "gzip": 18499
     },
     "react.production.min.js (NODE_PROD)": {
       "size": 9195,
       "gzip": 3614
     },
     "React-dev.js (FB_DEV)": {
-      "size": 73423,
-      "gzip": 18751
+      "size": 74104,
+      "gzip": 18972
     },
     "React-prod.js (FB_PROD)": {
       "size": 36836,
       "gzip": 9248
     },
     "ReactDOMStack-dev.js (FB_DEV)": {
-      "size": 496986,
-      "gzip": 118763
+      "size": 497667,
+      "gzip": 118999
```

Comment:
IMO we should be updating this file for when a PR is ready, as it updates master so it has the latest sizes – which is great for indicating when we've maybe let DEV code slip in unexpectedly.